packages/validator-js/test/validator-js.test.ts

```diff
@@ -1,8 +1,8 @@
 import { DataTypes, Model, Sequelize } from '@sequelize/core';
 import { Attribute } from '@sequelize/core/decorators-legacy';
+import { IsLowercase } from '@sequelize/validator.js';
```

Comment:
If you undo the changes in this file, the linter should pass without disabling `import/order`